test/models/study_file_test.rb

```diff
@@ -60,7 +60,7 @@ def setup
       taxon_id: Taxon.new.id,
       expression_file_info: ExpressionFileInfo.new(
         units: 'bad_value',
-        library_construction_protocol: 'Mars-seq',
+        library_preparation_protocol: 'Mars-seq',
```

Comment:
To make this validation issue more obvious:

```suggestion
        library_preparation_protocol: 'Mars-seq', # Incorrect case
```